Rust/Rust.sublime-syntax

```diff
@@ -6,7 +6,7 @@ file_extensions:
   - rs
 scope: source.rust
 variables:
-  identifier: '(?:[[:alpha:]][_[:alnum:]]*|_[_[:alnum:]]+)'
+  identifier: '(?:(?:[[:alpha:]][_[:alnum:]]*|_[_[:alnum:]]+)(?![_[:alnum:]]))' # include a negative lookahead at the end to ensure all possible characters are consumed, to prevent catastrophic backtracking
```

Comment:
Good point, I just somehow didn't think of it, because I'm not used to the `[[:alpha:]]` character class ;)